gradle/libs.versions.toml

```
@@ -20,6 +22,9 @@ accompanist-placeholder = { module = "com.google.accompanist:accompanist-placeho
 accompanist-swiperefresh = { module = "com.google.accompanist:accompanist-swiperefresh", version.ref = "accompanist" }
 accompanist-testharness = { module = "com.google.accompanist:accompanist-testharness", version.ref = "accompanist" }
 
+airbnb-showkase = { group = "com.airbnb.android", name = "showkase", version.ref = "showkase" }
```

```suggestion
airbnb-showkase = { module = "com.airbnb.android:showkase", version.ref = "showkase" }
```
Let's use `module` as much as possible (I tried to do it on the whole file, but not on `androidx-activity-compose` it seems)